### PR TITLE
ROX-17766, ROX-17780: upgrade test and psps

### PR DESCRIPTION
**Pull request by dashrews78**
## Description

The issue at hand was that recently all the gke clusters got bumped to `v1.25` of kubernetes meaning that pod security policies were no longer valid.  Since this test deploys old versions of software WITH old tooling to match it became problematic to get the old versions to deploy without pod security polices.  So I changed it from deploying with the deploy scripts to have the old versions of roxctl generate helm so I could use helm install and thus pass the flag for helm to ignore the pod security policies.

## Checklist
- [ ] Investigated and inspected CI test results
~- [ ] Unit test and regression tests added~
~- [ ] Evaluated and added CHANGELOG entry if required~
~- [ ] Determined and documented upgrade steps~
~- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs]~(https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

Purely fixing the test.

### CODE REVIEW

File: sensor/upgrader/upgradectx/context.go [hunk 1]
```diff
@@ -345,3 +352,8 @@ func (c *UpgradeContext) ListCurrentObjects() ([]*unstructured.Unstructured, err
 
 	return objects, nil
 }
+
+// IsPodSecurityEnabled returns whether or not pod security polices are enabled for this cluster
+func (c *UpgradeContext) IsPodSecurityEnabled() bool {
+	return c.podSecurityPoliciesSupported
```
Comment (porridge) [inline sensor/upgrader/upgradectx/context.go:358]: This field seems to be [set](https://github.com/stackrox/stackrox/pull/6505/files#diff-acc1578eababf29f1daa395679380b56b089fa2d0b0fe30ee316ffce9917bb1fR110) based on `OrderedBundleResourceTypes` which according to the comment will contain PSPs for all eternity, so I cannot see how this function could ever return anything else than `true`?

File: tests/upgrade/lib.sh [hunk 2]
```diff
@@ -85,20 +85,34 @@ deploy_earlier_central() {
     info "Deploying: $EARLIER_TAG..."
 
     if is_CI; then
-        gsutil cp "gs://stackrox-ci/roxctl-$EARLIER_TAG" "bin/$TEST_HOST_PLATFORM/roxctl"
+        make cli
```
Comment (connorgorman) [inline tests/upgrade/lib.sh:88]: This can just me `make cli`

**Review by connorgorman — APPROVED** [review]
Looks good

**Pull request merged**